handlers.go

```diff
@@ -651,7 +651,7 @@ func (s *Server) importFile(ctx context.Context, dserv ipld.DAGService, fi io.Re
 
 var noDataTimeout = time.Minute * 10
 
-func (cm *ContentManager) addDatabaseTrackingToContent(ctx context.Context, cont uint, dserv ipld.NodeGetter, bs blockstore.Blockstore, root cid.Cid, cb func(int64)) error {
+func (cm *ContentManager) addDatabaseTrackingToContent(ctx context.Context, cont uint, dserv ipld.NodeGetter, bs blockstore.Blockstore, root cid.Cid, cb func(int64)) (err error) {
```

Comment:
named returns are the devil, shame upon you and your family, 6 generations back!

Reply:
first of all i love the devil second of all how else are you supposed to return errors from defers

Reply:
at the very least dont name the out parameter error `err`, name it `outErr` or something if you absolutely must commit such sacrilege.

Reply:
kk it's gone